.dev/tests/php/bootstrap.php

```diff
@@ -1,19 +1,38 @@
 <?php
 
-$_tests_dir = getenv( 'WP_TESTS_DIR' );
+// Require dependencies.
+require_once dirname( dirname( dirname( dirname( __FILE__ ) ) ) ) . '/vendor/wp-phpunit/wp-phpunit/__loaded.php';
+require_once dirname( dirname( dirname( dirname( __FILE__ ) ) ) ) . '/vendor/yoast/phpunit-polyfills/phpunitpolyfills-autoload.php';
 
-if ( ! $_tests_dir ) {
+// If we're running in WP's build directory, ensure that WP knows that, too.
+if ( 'build' === getenv( 'LOCAL_DIR' ) ) {
+	define( 'WP_RUN_CORE_TESTS', true );
+}
 
-	$_tests_dir = '/var/www/html/wordpress-tests-lib';
+// Determine the tests directory (from a WP dev checkout).
+// Try the WP_TESTS_DIR environment variable first.
+$_tests_dir = getenv( 'WP_TESTS_DIR' );
 
+// Next, try the WP_PHPUNIT composer package if we're inside wp-env.
+if ( ! $_tests_dir && 'tests-mysql' === getenv( 'WORDPRESS_DB_HOST' ) ) {
+	$_tests_dir = getenv( 'WP_PHPUNIT__DIR' );
 }
 
-if ( ! file_exists( $_tests_dir . '/includes/functions.php' ) ) {
-
-	echo "Could not find $_tests_dir/includes/functions.php, have you run bin/install-wp-tests.sh ?" . PHP_EOL;
+// See if we're installed inside an existing WP dev instance.
+if ( ! $_tests_dir ) {
+	$_try_tests_dir = dirname( __FILE__ ) . '/../../../../../tests/phpunit';
+	if ( file_exists( $_try_tests_dir . '/includes/functions.php' ) ) {
+		$_tests_dir = $_try_tests_dir;
+	}
+}
+// Fallback.
+if ( ! $_tests_dir ) {
+	$_tests_dir = rtrim( sys_get_temp_dir(), '/\\' ) . '/wordpress-tests-lib';
+}
```

Comment:
Smart. Is this related to the new `"wp-phpunit/wp-phpunit"` package?

Reply:
@AnthonyLedesma Yeah, this is just to maintain backwards compatibility. We used to install WordPress and the test lib into the `tmp` directory, and that's how I still have it setup locally so this allows that to still work as it was previously.